IS7024/Pages/LudlowGarage.cshtml.cs

```diff
@@ -34,7 +34,7 @@ public void OnGet()
                 IList<string> validationEvents = new List<string>();
                 if (jsonObject.IsValid(schema, out validationEvents))
                 {
-                    var rootObject = RootObject.FromJson(jsonString);
+                    RootObject rootObject = RootObject.FromJson(jsonString);
```

Comment:
I think using the actual type here makes it easier for quick reference, clear and descriptive